### PR TITLE
Add submodule and read question details from MD files

### PR DESCRIPTION
**Pull request by ShakaibKhan**

### CODE REVIEW

**Comment by ShakaibKhan** [comment]
rn this is what I am thinking and you can have the questions as json object

**Comment by ShakaibKhan** [comment]
gonna start making more questions here https://github.com/ShakaibKhan/byteclub-questions

File: app/service/QuestionService.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    public constructor(id : string, title : string, description:string, testCases:TestCases) {
+        this.id = id;
+        this.title = title;
+        this.description = description;
+        this.testCases = testCases
+    }
+}
+
+class TestCases {
```
Comment (sparksflyupwards) [inline app/service/QuestionService.ts]: Can we move these classes to their own files and export them if we think they might be needed elsewhere

Comment (ShakaibKhan) [reply app/service/QuestionService.ts]: yeah I can move these to interfaces

**Comment by ShakaibKhan** [comment]
merging this as I can get question objects and now I need to create evaluation logic. A little concerned about the format of the questions and if it might be better to use regex to find these strings

**Pull request merged**